src/updateCode/update.ts

```diff
@@ -15,9 +15,8 @@ const throttle = pThrottle({
   interval: 2500,
 })
 
-const webhookPattern = new RegExp(
-  `^webhooks-\\d+|[[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}]-lambda-${ENV}$`
-)
+const webhookPattern = new RegExp(`^webhooks-(\\d+|app\\d+)-lambda-${ENV}$`)
+// const webhookPattern = new RegExp(`^webhooks-\\d+-lambda-${ENV}$`)
```

Comment:
*nit* we can remove this comment. Won't hold up the PR since its the only change to the PR

Reply:
Updated